### PR TITLE
Added some extra formatting

### PR DESCRIPTION
**Pull request by SilverTactic**

### CODE REVIEW

**Comment by DanielCranham** [comment]
What does this even improve?

**Comment by Zarel** [comment]
Hm, apparently it removes "a" when grammatically incorrect, and it uses the correct "winner" or "winners" depending on number of winners, instead of just displaying "winner(s)".

In other words, grammatical corrections. Actually not too bad an idea, but if you could actually indent your code properly... :|

**Comment by SilverTactic** [comment]
Welp. =|
Could you tell me what exactly is improper? The only issue I see is the extra space before the var firstPart thing.

File: chat-plugins/wifi.js [hunk 1]
```diff
@@ -150,10 +150,10 @@ var LotteryGiveAway = (function () {
 		this.prize = options.prize;
 		this.maxwinners = options.maxwinners;
 		this.joined = {}; // userid: 0
-
+                var firstPart = this.prize[0];
```
Comment (ascriptmaster) [inline chat-plugins/wifi.js]: Use tabs, not spaces.

**Comment by SilverTactic** [comment]
Alright, I guess I'll work on that. So um.. do I make a new pull or is there a way to edit this one?

**Comment by ascriptmaster** [comment]
Make your changes on your local client (make sure you're in the correct branch!), then use `git commit --amend` instead of `git commit` as normal. Then, when you push your repository, add a `-f` flag at the end to force push.

**Comment by SilverTactic** [comment]
I see. But that would pretty much be the same as making a new pull, wouldn't it?

**Comment by ascriptmaster** [comment]
The pull request updates automatically when you push to the branch you're requesting a pull from.

**Comment by SilverTactic** [comment]
Ah okay, I'll do that then. So it would be git push -f or something after committing? (Sorry, not too good with git).

**Comment by ascriptmaster** [comment]
`git push [remote name] [branch name] -f`

Your remote name is probably `origin`; that's what git sets it to when you do a `git clone`.

Your branch name in this case is `patch-3`, according to the pull request.

So you'd probably type `git push origin patch-3 -f`

**Comment by SilverTactic** [comment]
Hmm should I just make a new pull request? ._.

**Comment by ascriptmaster** [comment]
If you make a new pull request, you'll probably have to make a new branch, and re-do all your changes... It's a big mess, especially when you get to bigger changes than this.

You might as well learn git anyways.

**Comment by Zarel** [comment]
You should also run npm (or at least read the Travis CI output).

**Pull request closed without merging**